### ISSUE TITLE
Cannot disable pos annotator while running lexical parse

### ISSUE THREAD

**Issue by Genius1237** [stanfordnlp/CoreNLP#941]
## TLDR
CoreNLP server runs the `pos` option in the pipeline even when it is instructed not to (both while launching the server and while sending the POST request)

## Description
Originally submitted as a different issue on the parser-user mailing, list [here ](https://mailman.stanford.edu/pipermail/parser-user/2019-August/003545.html), Chris Manning suggest that CoreNLP be run without `pos` as an option.

The CoreNLP server was run with the following command
```
java -mx500m -Xms2048m  -Xmx2048m -cp "./*" edu.stanford.nlp.pipeline.StanfordCoreNLPServer -annotators tokenize,ssplit,parse
```

With the server running, requests were made to the server via different frontends(NLTK, Web, StanfordCoreNLPCLient). All these just end up sending a POST request to ```http://localhost:9000/```. For example, StanfordCoreNLPClient was run as follows
```
java -mx500m -Xms2048m  -Xmx2048m -cp "./*" edu.stanford.nlp.pipeline.StanfordCoreNLPClient -host localhost -port 9000 -annotators tokenize,ssplit,parse
```
An input string was then given. 

The line in the server log on getting the POST request says
```
[pool-1-thread-1] INFO CoreNLP - [/127.0.0.1:51882] API call w/annotators tokenize,ssplit,pos,parse
```
How can the lexical parser option be used without the pos stage being run?

**Comment by Genius1237** [comment]
I just built the package and confirmed that I was able to get this to work.

Thanks for your help. I will close the issue.
